_posts/0000-01-02-JHuberman.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+This is taking longer than I thought
+Use the left arrow to go back!
```

Comment:
Never use something when nothing will do
An insult has the stature of its source.